datasync/chaindatafetcher/kafka/kafka.go

```diff
@@ -60,11 +60,23 @@ func NewKafka(conf *KafkaConfig) (*Kafka, error) {
 		return nil, err
 	}
 
-	return &Kafka{
+	kafka := &Kafka{
 		config:   conf,
 		producer: producer,
 		admin:    admin,
-	}, nil
+	}
+
+	err = kafka.CreateTopic(conf.getTopicName(EventBlockGroup))
+	if err != nil {
+		logger.Warn("creating a block group is failed", "err", err)
```

Comment:
This routine is for checking if the topic is already created or not?
Why do you need this check routine?

```suggestion
		logger.Warn("already created a block group is failed", "err", err)
```